spec/remote/reports/github_api_client_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'remote_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.